common/src/main/java/com/microsoft/identity/common/internal/broker/MicrosoftAuthClient.java

```diff
@@ -140,4 +165,39 @@ private String getCurrentActiveBrokerPackageName(@NonNull final Context context)
 
         return null;
     }
+
+    /**
+     * Returns the inactive installed authenticator.
+     * - If CP is the broker, this will return authenticator.
+     * - If authenticator is the broker, this will return CP.
+     *
+     * @param context
+     * @return package name of the installed inactive broker app, if there's any.
+     */
+    private String getInactiveBrokerPackageName(@NonNull final Context context) {
+        final String methodName = ":getInactiveBrokerPackageName";
+        String currentBroker = getCurrentActiveBrokerPackageName(context);
+
+        final String inactiveBrokerPackageName;
+        if (currentBroker.equalsIgnoreCase(AuthenticationConstants.Broker.AZURE_AUTHENTICATOR_APP_PACKAGE_NAME)) {
```

Comment:
This looks to be a possible NPE... `getCurrentActiveBrokerPackageName()` will return `null` if there is no active broker -- flipping this check around a comparing against the constant would avoid this....

so

```java
if (AuthenticationConstants.Broker.AZURE_AUTHENTICATOR_APP_PACKAGE_NAME.equalsIgnoreCase(currentBroker)) {
    // ...
}
```

The same would apply to the `else if` clauses